File: exercicios em Python/q24.py
import os
print("\nEste programa calcula o faturamento mensal de um mercado!")
n = int(input("\nDigite o numero de mercadorias diferentes que voce possui: "))
i = 1
fat = 0
while i <= n:
        print("\nDigite o numero de unidades vendidas da ",i," mercadoria: ")
        num = int(input(""))
        print("\nDigite o preco da ",i," mercadoria: ")
        p = int(input(""))
        fat = (num * p) + fat
        i = i + 1

print("\nO faturamento mensal e de ",fat,".")
print("\nObriagado por utilizar nosso sistema!")
os.system("pause")

File: exercicios em C/q24.cpp
#include <stdio.h>

int main(void){
	int i, n, fat, p, num;
	fat = 0;
	printf("\nEste programa calcula o faturamento mensal de um mercado!");
	printf("\nDigite o numero de mercadorias diferentes que voce possui: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		printf("\nDigite o numero de unidades vendidas da %d mercadoria: ", i);
		scanf("%d", &num);
		printf("\nAgora digite o preco da %d mercadoria: ", i);
		scanf("%d", &p);
		fat = (num * p) + fat;
	}
	printf("\nO faturamento mensal e de %d.", fat);
	return 0;
}
